src/components/content-tab-overview.tsx

```diff
@@ -100,23 +109,29 @@ function CustomizeSection( {
 		{
 			label: __( 'Customizer' ),
 			icon: edit,
-			onClick: () => getIpcApi().openSiteURL( selectedSite.id, '/wp-admin/customize.php' ),
+			onClick: () => {
+				void getIpcApi().openSiteURL( selectedSite.id, '/wp-admin/customize.php' );
```

Comment:
Would it be equivalent in all these cases of onClick to make an async/await call?
I don't see any errors in my linter indicating that there is an unhandled error for this call.

```suggestion
			onClick: async () => {
				await getIpcApi().openSiteURL( selectedSite.id, '/wp-admin/customize.php' );
```

Reply:
Yep, this would also satisfy the linter. If you think that's preferable here, do you mind elaborating on why and when you think `void` is suitable compared to `async`/`await`?

Reply:
I used "void" in the past when defining the return type while declaring a function. It's the first time I've seen its usage when calling a function, so I'm not accustomed to it.
Using `async/await` feels more natural to me, as we are not overriding the return type of the function.

Reply:
Using it as an operator is definitely esoteric, even if it's [perfectly valid](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void). I'm guessing the ESLint authors chose `void` for this purpose since it doesn't change the functionality of the code, and it's relatively succinct. 

> Using async/await feels more natural to me

If we land this PR, I think we'll need to get accustomed to defaulting to `void` when we don't care about the return value or executing things in sequence. `async/await` will work the same way in many cases, but not always. 

> as we are not overriding the return type of the function

None of the `void` operators in this PR change the return type of any of the functions.